node/querystring.ts

```diff
@@ -276,7 +276,15 @@ const noEscape = new Int8Array([
  * replaces encodeURIComponent()
  * @see https://www.ecma-international.org/ecma-262/5.1/#sec-15.1.3.4
  */
-function qsEscape(str: string): string {
+// deno-lint-ignore no-explicit-any
+function qsEscape(str: any): string {
```

Comment:
Is it possible to make this type `unknown`?

Reply:
:+1: https://github.com/denoland/deno_std/pull/1522/commits/fd57b8a781787a36d28974169a1a687ac2ee466a